src/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import "../styles/App.css";
+import Search from "./Search";
+import SearchResults from "./searchResults";
+
+function App() {
+  const [searchResults, setSearchResults] = useState([]);
+  return (
+    <div className="App">
+      <img
+        className="nasa-logo"
+        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
+        alt="nasaLogo"
+      />
+      <Search setSearchResults={setSearchResults} />
+      <br />
+      <br />
```

Comment:
Why is `<br>` being used? If its layout concern using CSS to achieve this instead of creating DOM elements would be much more appropriate

Reply:
Removed <br> here and used CSS instead.